plugins/version-control-systems/git/src/main/kotlin/Git.kt

```diff
@@ -129,7 +128,7 @@ class Git internal constructor() : VersionControlSystem(GitCommand) {
     override fun getVersion() = GitCommand.getVersion()
 
     override fun getDefaultBranchName(url: String): String {
-        val refs = Git.lsRemoteRepository().setRemote(url).callAsMap()
+        val refs = org.eclipse.jgit.api.Git.lsRemoteRepository().setRemote(url).callAsMap()
```

Comment:
As an idea, how about importing it as `JGit` instead?

Reply:
Good idea, done.